### PR TITLE
Support 9.4 and 9.5

### PR DESCRIPTION
**Pull request by solidsnack**

### CODE REVIEW

File: src/remove_duplicate_single_line_statements.py [hunk 1]
```diff
@@ -1,8 +0,0 @@
-import sys
```
Comment (solidsnack) [inline src/remove_duplicate_single_line_statements.py:1]: Why I removed this: didn't seem to break the build not to have it; and less is more.

Comment (posix4e) [inline src/remove_duplicate_single_line_statements.py:1]: Love that this is not needed anymore. As long as bindgen can run we are good.

File: util/generate_bindings [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+cat > tmp/pgheaders.h <<\EOF
+#include <stdarg.h>
+#include "postgres.h"
+#include "fmgr.h"
+#include "utils/builtins.h"
+#include "utils/json.h"
+#include "utils/lsyscache.h"
+#include "replication/output_plugin.h"
+#include "replication/logical.h"
+#include "replication/logical.h"
```
Comment (posix4e) [inline util/generate_bindings]: Why do we have this duplicate line?

**Comment by posix4e** [comment]
Looking good mostly. I'm curious how we want to test 9.5. Is it enough to test it on osx, and keep 9.4 on linux?

**Pull request merged**